steve/util.py

```diff
@@ -378,10 +378,14 @@ def out(*output, **kw):
     sys.stdout.write(output + '\n')
 
 
-def load_json_files(config):
+def load_json_files(config, path=None):
```

Comment:
I don't want multiple data directories. The problem is that more than just the webeditor expected the first element from the tuples to be filenames starting with "json/" and I would have had to deal with those usages at this point as well, which I did not want to do. Hence the backward compatible default (for now).